pallets/gear-program/src/migrations/v13_split_instrumented_code_migration.rs

```diff
@@ -62,24 +63,19 @@ impl<T: Config> OnRuntimeUpgrade for MigrateSplitInstrumentedCode<T> {
             log::info!("🚚 Running migration from {onchain:?} to {update_to:?}, current storage version is {current:?}.");
 
             v12::CodeStorage::<T>::drain().for_each(|(code_id, instrumented_code)| {
-                // 1 read for instrumented code, 1 write for instrumented code and 1 write for code metadata
-                weight = weight.saturating_add(T::DbWeight::get().reads_writes(1, 2));
+                // 1 read for instrumented code and 1 write for code metadata
+                weight = weight.saturating_add(T::DbWeight::get().reads_writes(1, 1));
 
+                // We skip writing instrumented code into storage, as it will be written in the next reinstrumentation
                 let code_metadata = CodeMetadata::new(
                     instrumented_code.original_code_len,
                     instrumented_code.code.len() as u32,
```

Comment:
instrumented code len should be only part of InstrumentationStatus::Instrumented()